plugins/modules/proxmox_cluster_join_info.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2025, Florian Paul Azim Hoberg (@gyptazy) <florian.hoberg@credativ.de>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: proxmox_cluster_join_info
+version_added: 1.0.0
+short_description: Retrieve the join information of the Proxmox VE cluster
+description:
+  - Retrieve the join information of the Proxmox VE cluster.
+author: Florian Paul Azim Hoberg (@gyptazy)
+extends_documentation_fragment:
+  - community.proxmox.proxmox.actiongroup_proxmox
+  - community.proxmox.proxmox.documentation
+  - community.proxmox.attributes
+  - community.proxmox.attributes.info_module
+"""
+
+EXAMPLES = r"""
+- name: List existing Proxmox VE cluster join information
+  community.proxmox.proxmox_cluster_join_info:
+    api_host: proxmox1
+    api_user: root@pam
+    api_password: "{{ password | default(omit) }}"
+    api_token_id: "{{ token_id | default(omit) }}"
+    api_token_secret: "{{ token_secret | default(omit) }}"
+  register: cluster_join
```

Comment:
Why not keep the longer name here?
```suggestion
  register: proxmox_cluster_join
```

Reply:
ack